### PR TITLE
add Amharic consent and cookie banner translations

### LINKED ISSUE #3286: Add translations for cookie and privacy banner for 4 live radio v1 services
**Is your feature request related to a problem? Please describe.**
We need to add the cookie / privacy banner translations for the 4live radio v1 services 
(awaiting translations for Tigrinya see: https://github.com/bbc/simorgh/issues/3411

**Describe the solution you'd like**
Add the translations to the service configs. Translations available on spreadsheet (pinned in ws-media slack channel or WSMEDIA-30)

- Afaan Oromoo
- Amharic
- Indonesian
- Korean

Check with UX  to ensure the length of the translations are suitable (not too long)

**Testing notes**
[Tester to complete]

Dev insight: Unlikely to need a tester due to simplicity

**Additional context**
WSMEDIA-30

### PR DESCRIPTION
**Pull request by jroebu14**
Partially resolves https://github.com/bbc/simorgh/issues/3286

**Overall change:** Adds Amharic translations for the consent and cookie banner

**Code changes:**

- Adds translations for consent and cookie banner to `src/app/lib/config/services/amharic.js`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

**Testing notes**
Visit http://localhost.bbc.com:7080/amharic/bbc_amharic_radio/liveradio.

Clear cookies to view the consent banner. Click `OK` to view the cookie banner.

Translations can be verified here https://docs.google.com/spreadsheets/d/1IE6dvHfOJhYTIhC0kg8aRGnaj3fEg1NH9jQwo5NuUDQ/edit#gid=0

### CODE REVIEW

File: src/app/lib/config/services/amharic.js [hunk 1]
```diff
@@ -105,16 +105,16 @@ const service = {
               'https://www.bbc.co.uk/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
           },
           international: {
-            first: 'We and our partners use technologies, such as ',
-            linkText: 'cookies',
+            first: 'እኛም ሆን ቴክኖሎጂያችንን የሚጠቀሙ አጋሮቻችን ለምሳሌም  ',
+            linkText: 'ኩኪዎችን የምንጠቀመው',
             last:
-              ', and collect browsing data to give you the best online experience and to personalise the content and advertising shown to you. Please let us know if you agree.',
+              ', ና መረጃዎችንም የምንሰብስበው የሚፈልጓቸውን መረጃዎችና ማስታወቂያዎች ቅድሚያ እንዲያገኙዋቸው በማሰብና አግልግሎታችንም የተሻለ እንዲሆን ነው።እባክዎ መስማማትዎን ያሳውቁን',
```
Comment (pjlee11) [inline src/app/lib/config/services/amharic.js]:
```suggestion
              ', ና መረጃዎችንም የምንሰብስበው የሚፈልጓቸውን መረጃዎችና ማስታወቂያዎች ቅድሚያ እንዲያገኙዋቸው በማሰብና አግልግሎታችንም የተሻለ እንዲሆን ነው።እባክዎ መስማማትዎን ያሳውቁን።',
```

**Review by sareh — APPROVED** [review]
👍 Looks good.

**Review by pjlee11 — APPROVED** [review]

**Comment by sareh** [comment]
Moved into 'ready for test' column since we do not have automated Chromatic QA tests for the various service translations for the Consent banner https://bbc.github.io/simorgh/?path=/story/containers-consent-banner--canonical

**Comment by jamesbhobbs** [comment]
> Moved into 'ready for test' column since we do not have automated Chromatic QA tests for the various service translations for the Consent banner https://bbc.github.io/simorgh/?path=/story/containers-consent-banner--canonical

We don't need chromatic qa for this. IMO, what does need manual testing though is that these strings are correct and against the right value.

**Comment by LilyL0u** [comment]
In the translation in the cookie banner, in the field 'Cookie banner content non-UK', on the website there is a :: symbol on the end of the last sentence (full stop), but in the translation spreadsheet, this sentence does not have a :: on the end. I don't think we can assume there should be what we think is the equivalent of a full stop on the end of a sentence in a language where we don't know the punctuation rules of that language. It might be a mistake that the translator left it off, or there could be a reason why the translator decided not to add the :: .

**Comment by aeroplanejane** [comment]
Yes, agreed. Let's assume that the spreadsheet is the source of truth, even though we can assume this requires a full stop - if editorial fail to specify the text with correct punctuation it is not on us to correct them and add punctuation. We will verify with editorial again before we go live as part of our pre-release plan.

**Comment by aeroplanejane** [comment]
Going forward let's use the spreadsheet as the source of truth - in this instance we think this is fine to go ahead.

**Pull request merged**